### ISSUE TITLE
Make filter panel more accessible

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6508]
On the search results page, we hide the classifier filters on smaller resolutions, revealing them in a panel.

We need to:

1. ensure that the keyboard focus is move to the panel, when it is opened by the user
2. ensure that the user cannot 'tab out' of the panel when it is open (e.g. users should be able to loop over the links in the panel

**Comment by nlhkabu** [comment]
I *think* we could probably approach this issue by making https://github.com/pypa/warehouse/blob/master/warehouse/static/js/warehouse/utils/bind-modal-keys.js more generic and applying it to both the modal and the tab panel.

@woodruffw - this is probably outside of my JS abilities - would you be able to look into this?
